app/configuration/FrontEndInfoConfiguration.scala

```diff
@@ -0,0 +1,10 @@
+package configuration
+
+import javax.inject.Inject
+import play.api.Configuration
+import viewsapi.FrontEndInfo
+
+class FrontEndInfoConfiguration @Inject ()(configuration: Configuration) {
```

Comment:
Wonder whether "...Info..." is the best word to use in the name? Seems a bit vague. Although can't really think of a better one.

Reply:
I'm not too pleased either, it was originally authenticationDetails or something like that but there's the api url and the stage so I tried to go for something generic.

Reply:
Fair enough